src/components/select/simple-select/simple-select.stories.mdx

```diff
@@ -652,6 +653,76 @@ The inline label can change to be top aligned at a breakpoint. Enable this by pa
   </Story>
 </Canvas>
 
+#### New designs validation
+
+<Preview>
```

Comment:
Same here too, storybook also crashes when clicking to open select list?

Reply:
That's an issue on master 👍🏻 https://github.com/Sage/carbon/issues/4878